ocaml/parsing/pprintast.ml

```diff
@@ -1847,10 +1862,22 @@ and jane_syntax_expr ctxt attrs f (jexp : Jane_syntax.Expression.t) =
     pp f "((%a)@,%a)" (jane_syntax_expr ctxt []) jexp
       (attributes ctxt) attrs
   else match jexp with
+  | Jexp_local x            -> local_expr ctxt f x
   | Jexp_comprehension x    -> comprehension_expr ctxt f x
   | Jexp_immutable_array x  -> immutable_array_expr ctxt f x
   | Jexp_unboxed_constant x -> unboxed_constant ctxt f x
 
+and local_expr ctxt f (lexp : Jane_syntax.Local.expression) =
+  match lexp with
+  | Lexp_local expr ->
+    pp f "@[<2>local_ %a@]" (expression ctxt) expr
```

Comment:
I think the pretty printer isn't being eager enough about putting parentheses around `local_` expressions, e.g. this:

```ocaml
(local_ 1) + (local_ 2)
```
gets rewritten to this:
```ocaml
local_ 1 + local_ 2
```

This does seem like a regression. Is it easy to maintain the previous behavior?

Reply:
This is clearly a bug, thanks